tests/tests/swfs/avm2/font_hasglyphs/Test.as

```diff
@@ -0,0 +1,63 @@
+﻿package {
+	public class Test {
+	}
+}
+
+trace("///new NotoSansRegular().hasGlyphs(\"abc\")");
trace(new NotoSansRegular().hasGlyphs("abc"));

```

Comment:
broken newline?

Reply:
Dunno how it wound up here but it's gone now.